.github/workflows/ci.yml

```diff
@@ -126,6 +126,107 @@ jobs:
             ${{ env.CHAIN_SPEC }}.json
             ${{ env.CHAIN_SPEC }}.json.state
 
+  integration-test:
+    #      name: ${{ matrix.test }}
+    name: integration-test
+    runs-on: ubuntu-22.04
+    needs: build_primary_binaries
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - test: bootstrap_demo_community.py --signer //Alice -w collective --batch-size 7 --waiting-blocks 5 --is-parachain --test cc-fee-payment
```

Comment:
I wanted to use an env var here, to remove redundancy. But this is not possible in the matrix definition unfortunately.

Reply:
ok